src/Middleware/HttpCache.php

```diff
@@ -107,15 +86,10 @@ public function __construct(
 
     public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
     {
-        if (!$this->enabled) {
-            return $handler->handle($request);
-        }
-
         $method = $request->getMethod();
         if (
-            !in_array($method, [Method::GET, Method::HEAD])
-            || $this->lastModified === null
-            && $this->etagSeed === null
+            !\in_array($method, [Method::GET, Method::HEAD])
```

Comment:
Seems we're using it wrong :( `Method` is part of router package.

Reply:
It's used in some middleware

Reply:
Yes and it's wrong...

Reply:
Then should we use strings instead?

Reply:
These constants are useful so I think we can move Method from router package to web and use strings in the router package instead.

Reply:
how about `yiisoft/http-utilities` package with `Method` class and `HeaderHelper` class

Reply:
@kamarton Did you mean the new `yiisoft/http-utilities` package?

Reply:
yes

Reply:
> how about yiisoft/http-utilities package with Method class and HeaderHelper class?

@samdark ?

Reply:
https://github.com/yiisoft/http. Other packages code is already updated.